src/store/analytics/providers/segment.ts

```diff
@@ -41,14 +43,25 @@ export const track = (
   options?: Record<string, any>,
   callback?: () => void
 ) => {
+  // Add generic track event context for every event
+  const propertiesWithContext = {
+    ...properties,
+    clientVersion: version
+  }
+
   if (NATIVE_MOBILE) {
-    const message = new TrackAnalyticsEvent(event, properties)
+    const message = new TrackAnalyticsEvent(event, propertiesWithContext)
     message.send()
   } else {
     if (!(window as any).analytics) {
       if (callback) callback()
       return
     }
-    ;(window as any).analytics.track(event, properties, options, callback)
+    ;(window as any).analytics.track(
```

Comment:
nbd, but noticed the semi-colon character in the beginning of this line which was there before too. maybe a mistype?

Reply:
nope actually, lines beginning with ( need it
https://stackoverflow.com/questions/7145514/whats-the-purpose-of-starting-semi-colon-at-beginning-of-javascript